core/node/base_token_adjuster/README.md

```diff
@@ -0,0 +1,13 @@
+# Base Token Adjuster
+
+This crate contains all the logic to handle ZK Chain with custom base tokens. It is used by other node layers to adjust
+the fees to be denominated in the chain's base token.
+
+## Overview
+
+The Base Token Adjuster:
+
+- Connect with external APIs to get the current price of the base token.
+- Persist the price of the base token in the database.
+- Upon request, adjust the fees to be denominated in the base token.
```

Comment:
This is not true, right? The component doesn't seem to do that.

Reply:
Correct. Updated the readme!